samples/tab-graph-toolkit/csharp/TabGraphToolkit/ClientApp/src/App.css

```diff
@@ -0,0 +1,11 @@
+.App {
+  text-align: left;
+}
```

Comment:
App "A" is in capital letter and other CSS are in different format.
Please use camel case.